frontend/src/metabase/metadata/components/SyncTableSchemaButton/SyncTableSchemaButton.tsx

```diff
@@ -0,0 +1,38 @@
+import { useEffect, useState } from "react";
+import { t } from "ttag";
+
+import { useSyncTableSchemaMutation } from "metabase/api";
+import { useToast } from "metabase/common/hooks";
+import { Button } from "metabase/ui";
+import type { TableId } from "metabase-types/api";
+
+export function SyncTableSchemaButton({ tableId }: { tableId: TableId }) {
+  const [started, setStarted] = useState(0);
+  const [syncTableSchema] = useSyncTableSchemaMutation();
+  const [sendToast] = useToast();
+
+  const handleClick = async () => {
+    const { error } = await syncTableSchema(tableId);
+
+    if (error) {
```

Comment:
Let's keep this no different than in other similar buttons. If this approach is better - let's use it everywhere.